### PR TITLE
Added solution HW-1.3

### PR DESCRIPTION
**Pull request by aborozdykh**

### CODE REVIEW

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 1]
```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a != b) & (c != d)) | ((a != c) & (b != d));
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BooleanExpression.java]: try to use binary operators

Comment (aborozdykh) [reply src/main/java/core/basesyntax/BooleanExpression.java]: Done.

File: src/main/java/core/basesyntax/BooleanExpression.java [hunk 2]
```diff
@@ -14,6 +14,6 @@
      * без использования конструкций if-else</p>
      */
     public boolean booleanExpression(boolean a, boolean b, boolean c, boolean d) {
-        return false;
+        return ((a != b) && (c != d)) || ((a != c) && (b != d));
```
Comment (dmytro-hryhoruk) [inline src/main/java/core/basesyntax/BooleanExpression.java]: try to use binary operators

Comment (aborozdykh) [reply src/main/java/core/basesyntax/BooleanExpression.java]: Done.

**Review by dmytro-hryhoruk — APPROVED** [review]

**Pull request closed without merging**